operator/v1/types_ingress.go

```diff
@@ -244,6 +244,7 @@ type EndpointPublishingStrategy struct {
 	// networking, and is not explicitly published. The user must manually publish
 	// the ingress controller.
 	// +unionDiscriminator
+	// +kubebuilder:validation:Required
```

Comment:
Elsewhere I see `+kubebuilder:validation:Required` paired with `+required` (I would guess for consistency across various tools), so we should probably use both markers here.

Reply:
Fixed

Reply:
Yeah, `+required` is for backwards compatibility/consistency. Both are supported